lib/Room.php

```diff
@@ -327,6 +340,12 @@ public function getPropertiesForSignaling(string $userId): array {
 			'sip-enabled' => $this->getSIPEnabled(),
 		];
 
+		if ($roomModified) {
+			$properties = array_merge($properties, [
+				'description' => $this->getDescription(),
+			]);
```

Comment:
This is not changing anything on the signaling?

Reply:
What do you mean?

Reply:
This is `getPropertiesForSignaling` but the data is not used in signaling, so we don't need to expose it?

Reply:
The new description will be relayed by the signaling server to the clients in the `roomlist->update` message. Currently we are not using the updated properties from the message, though, but I think it is better to send it for consistency with the rest of the update messages.